doc/sphinx/source/users-guide/base/forloops.rst

```diff
@@ -0,0 +1,113 @@
+:title: Users Guide: For loops
+
+.. _ug-forloops:
+
+For-loops
+=========
+
+*For-loops* are Chapel's means of expressing serial iteration as
```

Comment:
Something about how our for-loops are more like Python or Rust than C or Java, in that they're range based instead of a glorified while loop.